siege_utilities/identifiers/uuid_generation.py

```
@@ -0,0 +1,93 @@
+        seed: Non-empty string — the canonical identifier input. Consumers are
+            responsible for seed-ladder logic (picking the best available
+            identifier at resolution time).
+
+    Returns:
+        The UUID5 derived from ``namespace`` and ``seed``.
+
+    Raises:
+        ValueError: if ``seed`` is empty.
+    """
+    if not seed:
+        raise ValueError("UUID5 seed must be non-empty")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)